src/main/webapp/styleNzinga.css

```diff
@@ -0,0 +1,245 @@
+#content {
+  margin-left: auto;
+  margin-right: auto;
+  width: 650px;
+}
+
+#location-container {
+  margin-top: 20px;
+}
+
+body,
+html {
+  height: 100%;
+}
+
+div {
+  margin: 0;
+}
+
+/* Add a black background color to the top navigation bar */
+.topnav {
+  overflow: hidden;
+  background-color: #e9e9e9;
+  text-align: center;
+}
+
+/* Style the links inside the navigation bar */
+.topnav a {
+  color: black;
+  text-align: center;
+  padding: 14px 16px;
+  text-decoration: none;
+  font-size: 17px;
+  display: inline-block;
+}
+
+/* Change the color of links on hover */
+.topnav a:hover {
+  background-color: #ddd;
+  color: black;
+}
+
+/* Style the "active" element to highlight the current page */
+.topnav a.active {
+  color: #00bcd4;
+}
+
+/* Style the search box inside the navigation bar */
+.topnav input[type='text'] {
+  float: right;
+  padding: 6px;
+  border: none;
+  margin-top: 8px;
+  margin-right: 16px;
```

Comment:
consider using shorthand [properties](https://google.github.io/styleguide/htmlcssguide.html#Shorthand_Properties)

Reply:
I'll fix in a later pr